geomesa-blobstore/geomesa-blobstore-core/src/main/java/org/locationtech/geomesa/blob/core/interop/GeoMesaBlobStore.java

```diff
@@ -0,0 +1,72 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+
+package org.locationtech.geomesa.blob.core.interop;
+
+import org.geotools.data.Query;
+import org.opengis.filter.Filter;
+import scala.Option;
```

Comment:
well we followed a slightly different pattern by returning an option of the blob's key (which is a simplefeature id). I can switch over to that pattern, but I will have to change a number of unit tests. I would prefer to return the blob's key as a string (or null) instead of the value